code/modules/power/solar.dm

```diff
@@ -220,7 +220,13 @@
 		if(istype(W, /obj/item/stack/sheet/glass) || istype(W, /obj/item/stack/sheet/rglass))
 			var/obj/item/stack/sheet/S = W
 			if(S.use(2))
-				glass_type = W.type
+				// Snowflake checks to make sure when the solar panel is deconstructed, it doesn't spawn cyborg glass.
```

Comment:
Can you use the `merge_type` value here instead?

Reply:
That is a much cleaner idea.